examples/console/main.go

```diff
@@ -232,33 +232,33 @@ func run() error {
 				ObservationPeriodTime:      observationPeriodTime,
 				ObservationPeriodLedgerGap: observationPeriodLedgerGap,
 				MaxOpenExpiry:              maxOpenExpiry,
-				EscrowAccountKey:           escrowAccountKey,
+				MultiSigAccountKey:         multiSigKey,
 			}
 		}
 		underlyingAgent = agentpkg.NewAgent(config)
 
-		tx, err := txbuild.CreateEscrow(txbuild.CreateEscrowParams{
+		tx, err := txbuild.CreateMultiSig(txbuild.CreateMultiSigParams{
 			Creator:        accountKey.FromAddress(),
-			Escrow:         escrowAccountKey.FromAddress(),
+			MultiSig:       multiSigKey.FromAddress(),
 			SequenceNumber: accountSeqNum + 1,
 			Asset:          txnbuild.NativeAsset{},
 		})
 		if err != nil {
-			return fmt.Errorf("creating escrow account tx: %w", err)
+			return fmt.Errorf("creating multisig account tx: %w", err)
```

Comment:
For similar reasons as in the SEP change, I think it is confusing to use the term multisig here, because the account is not a multisig account in this moment, it's a singlesig account. I think we should probably try to omit the term multisig, simply replacing escrow with nothing, in the vast majority of places. Within the SDK we should find that the SDK only cares about a single account for each participant.

Where that might get a bit fuzzy is in the examples because in there each participant has two accounts. I think we can refer to that other account as the `fundingAccount`. Or `personalAccount`.

Reply:
are you thinking we should change `type MultiSigAccount` to `type Account`?

A problem with that I can foresee is the confusion between the non-multisig account and the multisig account, since the channel has to be aware of the non-multisig accounts. ie. `Channel.localSigner` and `Channel.remoteSigner` 

if we removed `multisig` from the `multisigAccount` fields and variable names, I think it would be pretty easy to conflate the non-multisig account field/variables with the multisig ones.
eg. `channel.localSigner` seems confusing to have next to `channel.localAccount`

though I agree with removing `multiSig` from the error strings, as you pointed out.

wdyt?

Reply:
I think some of the confusion is that in the examples, and in the txbuild.CreateEscrow, we treat the localSigner as the sponsor and the signer, as this account that does both of those things. I think we should refer to that account as the sponsoring account and treat the signer as a different key. I think if we do that the channel will have an account that it is using, then anything outside the channel, like the sponsoring account, is out of scope for most of the code.

Reply:
I think this will be less confusing if we get rid of CreateEscrow. It really doesn't need to be part of the SDK.

Reply:
we could change `channel.localSigner` to `channel.localFundingSigner` though 🤔

```
type Channel struct {
   ...
	initiator             bool
	localAccount  *Account
	remoteAccount *Account

	localFundingSigner  *keypair.Full
	remoteFundingSigner *keypair.FromAddress
}
```
> Within the SDK we should find that the SDK only cares about a single account for each participant.

what's the main reason for this again?

and could we do in a separate PR?  Seems to me to be a separate simplification, since this PR was for removing `escrow` only

Reply:
> `channel.localSigner` to `channel.localFundingSigner`

I don't think the distinction is meaningful. The local signer is the signer of the escrow account. That's actually all the channel needs to know, so `localSigner` is still a good name.

The fact that it is also the signer of the sponsoring account is happenstance and shouldn't be required.

I understand the desire to merge this as is, but there's still places where the term multisig is used where it is confusing as the account isn't a multisig account. Unfortunately, just replacing escrow with multisig doesn't retain the same clarity and I think we need to do more here.

Reply:
ok I'm in for doing here 😄 . Are you thinking we should change `type MultisigAccount` to `type Account`?

Reply:
We might need to iterate on this or work with it to try and figure it out. But I think target `type Account`, and I think in most of the SDK this should mean referring to the escrow account as simply the account in most places. i.e. localAccount and remoteAccount (different to now), and localSigner and remoteSigner (as they are now). It's probably too much to drop CreateEscrow in this PR, but if there's confusion with other accounts, we could use the term "sponsorAccount" in those places?

Reply:
one last thought before I make this change 😄  

what if we used a different word rather than `Account`? Account by itself to me is pretty generic. Eg. there are methods and variables we have atm that use the word `account`, which is not a payment channel account. That may be confusing with this new type that is a special kind of Account. Instead of changing those to be `sponsorAccount` or another type of `account`, we could change this new type to have a different name.
eg. `initAccounts`, `createAccount` in the integration tests, some other spots we use the word `account` generically as a variable

what about `type Participant`?

Reply:
That could work? We use the `<Noun>Details` pattern elsewhere in the SDK, maybe similar, to indicate this is just a data bag and isn't an actor or similar.

```go
type ParticipantDetails struct {
	Account *keypair.FromAddress
	Signer *keypair.FromAddress
}
```

I wonder if this could get confusing because the channel will have an account, and we would say the participant has an account, but the participants "account" is probably one way to talk about the sponsor's "account".

It's difficult to imagine this stuff without trying it out.